### ISSUE TITLE
AzureML Workbench

### ISSUE THREAD

**Issue by ghost** [Azure/MachineLearningSamples-cntk#4]
1.  Can we run this locally using AzureML Workbench?
By locally I mean run Jupyter notebook+CNTK locally on the client machine.
2.  Can I simply add project + include these scipts inside windows version of AzureML?

**Comment by hning86** [comment]
Yes, but you will need to pip install the Windows version of CNTK library first.
